modules/builtin/src/content-types/image.js

```diff
@@ -41,7 +41,11 @@ module.exports = {
       return
     }
 
-    const link = utils.formatURL(formData.BOT_URL, formData.image)
+    let imageUrl = formData.image
+    if (utils.isBpUrl(imageUrl)) {
+      imageUrl = '/' + imageUrl
+    }
+    const link = utils.formatURL(formData.BOT_URL, imageUrl)
```

Comment:
Should this logic be added to audio and video content-types?

Reply:
Oh, maybe it should

Reply:
done